frontend/src/patientApp/PatientApp.tsx

```diff
@@ -73,38 +73,37 @@ const PatientApp = () => {
     <Page>
       <PatientHeader />
       <PatientLinkURL404Wrapper plid={plid}>
-        <Router basename={`${process.env.PUBLIC_URL}/pxp`}>
-          <Switch>
-            <Route path="/" exact component={TermsOfService} />
-            <Route path="/terms-of-service" component={TermsOfService} />
-            <Route path="/birth-date-confirmation" component={DOB} />
-            <GuardedRoute
-              auth={auth}
-              path="/patient-info-confirm"
-              component={PatientProfileContainer}
-            />
-            <GuardedRoute
-              auth={auth}
-              path="/patient-info-edit"
-              component={PatientFormContainer}
-            />
-            <GuardedRoute
-              auth={auth}
-              path="/patient-info-symptoms"
-              component={AoEPatientFormContainer}
-            />
-            <GuardedRoute
-              auth={auth}
-              path="/success"
-              component={PatientLanding}
-            />
-            <GuardedRoute
-              auth={auth}
-              path="/test-result"
-              component={TestResult}
-            />
-          </Switch>
-        </Router>
+        <Routes>
+          <Route path="/" element={<TermsOfService />} />
+          <Route path="terms-of-service" element={<TermsOfService />} />
+          <Route path="birth-date-confirmation" element={<DOB />} />
+          <Route
+            path="patient-info-confirm"
+            element={
+              <GuardedRoute auth={auth} element={<PatientProfileContainer />} />
+            }
+          />
+          <Route
+            path="patient-info-edit"
+            element={
+              <GuardedRoute auth={auth} element={<PatientFormContainer />} />
+            }
+          />
+          <Route
+            path="patient-info-symptoms"
+            element={
+              <GuardedRoute auth={auth} element={<AoEPatientFormContainer />} />
+            }
+          />
```

Comment:
I think we might be able to kill this route, AFAIK we removed the ability for patients to fill out the AoE.

Reply:
That's true, but I think @zdeveloper may be working on cleaning these routes out in another PR IIRC.